providers/common/io/src/airflow/providers/common/io/xcom/backend.py

```diff
@@ -51,13 +51,21 @@ def _get_compression_suffix(compression: str) -> str:
     """
     Return the compression suffix for the given compression.
 
-    :raises ValueError: if the compression is not supported
+    :raises ValueError: if the compression is not supported or no compression is available
     """
-    for suffix, c in fsspec.utils.compressions.items():
-        if c == compression:
-            return suffix
+    available = fsspec.available_compressions()  # returns list
```

Comment:
```suggestion
    available: list = fsspec.available_compressions()
```

nit: a type hint might be cleaner than a comment for understanding the type of data returned from the function